### ISSUE TITLE
Not able to listen to audio.Does it work only for premier accounts ?

### ISSUE THREAD

**Issue by revanthgopi** [swapagarwal/medium-play#3]
@swapagarwal I am not able to listen to the audio..

**Comment by revanthgopi** [comment]
I have gone through the source code.. and tried to see the response for the below get request
https://medium.com/p/48743e4c761f/notes/

`])}while(1);</x>{"success":true,"payload":{"value":[],"post":{"id":"48743e4c761f","versionId":"","creatorId":"92b453b29f6b","creator":{"userId":"92b453b29f6b","name":"iB Hubs","username":"ibhubs","createdAt":1479810368907,"lastPostCreatedAt":1514611170738,"imageId":"1*L66ZxpT9QdnocxbxXo1xpQ.jpeg","backgroundImageId":"","bio":"iB Hubs is committed to developing the culture of innovation and entrepreneurship in every nook and corner of India.","twitterScreenName":"","facebookAccountId":"","allowNotes":1,"mediumMemberAt":0,"isNsfw":false,"type":"User"},"homeCollectionId":"","title":"","detectedLanguage":"en","latestVersion":"52ec09ea8ce3","latestPublishedVersion":"52ec09ea8ce3","hasUnpublishedEdits":false,"latestRev":63,"createdAt":1514435385422,"updatedAt":1514439724307,"acceptedAt":0,"firstPublishedAt":1514436505053,"latestPublishedAt":1514436505053,"experimentalCss":"","displayAuthor":"","coverless":true,"slug":"ib-hubs-student-challenge","translationSourcePostId":"","translationSourceCreatorId":"","isApprovedTranslation":false,"inResponseToPostId":"","inResponseToRemovedAt":0,"isTitleSynthesized":false,"allowResponses":true,"importedUrl":"","importedPublishedAt":0,"visibility":0,"uniqueSlug":"ib-hubs-student-challenge-48743e4c761f","license":0,"inResponseToMediaResourceId":"","canonicalUrl":"","approvedHomeCollectionId":"","newsletterId":"","webCanonicalUrl":"","mediumUrl":"","migrationId":"","notifyFollowers":true,"notifyTwitter":false,"isSponsored":false,"isRequestToPubDisabled":false,"notifyFacebook":false,"responseHiddenOnParentPostAt":0,"isSeries":false,"isSubscriptionLocked":false,"seriesLastAppendedAt":0,"audioVersionDurationSec":0,"sequenceId":"","isNsfw":false,"isEligibleForRevenue":false,"isBlockedFromHightower":false,"deletedAt":0,"lockedPostSource":0,"hightowerMinimumGuaranteeStartsAt":0,"hightowerMinimumGuaranteeEndsAt":0,"type":"Post"},"highlights":[],"references":{"User":{"92b453b29f6b":{"userId":"92b453b29f6b","name":"iB Hubs","username":"ibhubs","createdAt":1479810368907,"lastPostCreatedAt":1514611170738,"imageId":"1*L66ZxpT9QdnocxbxXo1xpQ.jpeg","backgroundImageId":"","bio":"iB Hubs is committed to developing the culture of innovation and entrepreneurship in every nook and corner of India.","twitterScreenName":"","facebookAccountId":"","allowNotes":1,"mediumMemberAt":0,"isNsfw":false,"type":"User"}}}},"v":3,"b":"32111-acac638"}`

This is the response and it doesn't have the expected `audioVersionUrl` there.. any clue whats going on ?

**Comment by revanthgopi** [comment]
On looking into the issue further.. I could see that only few articles have this `audioVersionUrl` in the get response.. so probably enabling the extension for only there articles would make more sense

**Comment by swapagarwal** [comment]
@revanthgopi Agreed, this should be enabled only when there's an audio version of that story available.
P.S. You don't require a medium membership for this to work!

**Comment by yashikagarg13** [comment]
@swapagarwal I would like to contribute to the project by fixing the above bug. Please point to your contribution.md file where I can see the process of contribution.

**Comment by harshlele** [comment]
is this issue still open?

**Comment by ivandev-81** [comment]
Hi, Is this still open? I want contribute
